docs/tutorials/visualization.rst

```diff
@@ -182,6 +182,34 @@ Now we specify the figure options before running the experiment for a second tim
 As can be seen in the figure, the different series generated by the experiment
 were styled differently according to the ``series_params`` attribute of ``figure_options``.
 
+By default, the supported figure options are ``xlabel``, ``ylabel``, ``xlim``, ``ylim``,
+``xval_unit``, ``yval_unit``, ``xval_unit_scale``, ``yval_unit_scale``, ``xscale``, ``yscale``,
+``figure_title``, and ``series_params``. The following T1 experiment provides examples to
```

Comment:
I think it would help to link to the `MplDrawer` API docs here so users can see details on what the options do.

Reply:
Can you please check 4116dd7 ?  I'm not sure if this is what you mean

Reply:
I was thinking something like,
```
see `:class:.MplDrawer` for details on how to set these options
```

Reply:
Sure. How's 051ecef ?